readme_template.md

```diff
@@ -461,33 +456,21 @@ devices under a variety of operating systems.
 ## Third-Party Hosts Managers
 
 * [Unified Hosts AutoUpdate](https://github.com/ScriptTiger/Unified-Hosts-AutoUpdate "Unified Hosts AutoUpdate") (for Windows): The Unified Hosts AutoUpdate package is purpose-built for this unified hosts project as well as in active development by community members. You can install and uninstall any blacklist and keep it automatically up to date, and can be placed in a shared network location and deployed across an organization via group policies. And since it is in active development by community members, your bug reports, feature requests, and other feedback are most welcome.
-
```

Comment:
Spacing between list element may actually be a good idea ...

Reply:
It's subjective for sure, but since I noticed it I thought I'd make a PR.

Reply:
I pushed a couple of other tweaks too which should improve things a little bit.

I'm fine with reverting the newlines if you all prefer them.